tests/constants/api_constants.ts

```diff
@@ -1 +1,11 @@
-export const LETTERS_ENDPOINT = 'letters';
+export const SUPPLIER_LETTERS = 'letters';
+export const SUPPLIER_API_URL = 'https://internal-dev.api.service.nhs.uk/nhs-notify-supplier/';
+export const SUPPLIER_API_URL_SANDBOX = 'https://internal-dev-sandbox.api.service.nhs.uk/nhs-notify-supplier';
```

Comment:
Are we wanting to run sandbox tests always against internal-dev-sandbox? For these tests I would've expected them to run against the PR's sandbox (or main)?